include/mbedtls/ssl_internal.h

```diff
@@ -487,6 +487,117 @@ static inline int mbedtls_ssl_safer_memcmp( const void *a, const void *b, size_t
     return( diff );
 }
 
+#if defined(MBEDTLS_SSL_SERVER_NAME_INDICATION)
+#include <ctype.h>
```

Comment:
The only reason why this header is used is because of calls to `isdigit()` and `isalpha()`, so this dependency could be removed.